src/expire.c

```diff
@@ -537,8 +537,12 @@ void expireGenericCommand(client *c, long long basetime, int unit) {
         addReply(c, shared.cone);
         return;
     } else {
-        setExpire(c,c->db,key,when);
+        setExpire(c,c->db,key,milliseconds);
         addReply(c,shared.cone);
+        /* Propagate as PEXPIREAT millisecond-timestamp */
+        robj *millisecondObj = createStringObjectFromLongLong(milliseconds);
```

Comment:
`millisecond_obj` for consistent naming schemes for local variables.

Reply:
Done.

To clarify, I copy-pasted this variable name from:
https://github.com/redis/redis/blob/92bde124caaedef0e1572765854b2deb9aa22a35/src/t_string.c#L125

I'm changing there too.